webxr/openxr/mod.rs

```diff
@@ -416,6 +419,9 @@ impl Device for OpenXrDevice {
                     ])],
             )
             .unwrap();
+        if let Some(sync) = sync {
```

Comment:
Don't we want this before we do the readback?

Reply:
idk, cc @asajeffrey ?

Reply:
I think you're right.